sspulltorefresh/src/main/java/com/simform/refresh/ViewExtensions.kt

```diff
@@ -0,0 +1,18 @@
+package com.simform.refresh
+
+import android.view.View
+
+val View.windowCoordinates: Pair<Int, Int>
+    get() {
+        val location = IntArray(2)
+        getLocationInWindow(location)
+        val x = location[0]
+        val y = location[1]
+        return x to y
+    }
+
+fun View.hasPoint(x: Int, y: Int): Boolean {
```

Comment:
Can you please define method description/purpose along with parameters?

Reply:
Added Java doc